requirements.txt

```diff
@@ -23,6 +23,7 @@ phpserialize==1.3.0
 kombu==3.0.35
 redis==2.10.5
 apscheduler==3.1.0
+requests==2.9.1
```

Comment:
You can then remove it from [dev-requirements.txt](https://github.com/c2corg/v6_api/blob/947c2284557a31564e121ed0082a348f882eaad1/dev-requirements.txt#L5).

And yes, it is correct. It should be listed under `requirements.txt`.